test/compilable/test7172.d

```diff
@@ -6,10 +6,6 @@ void main()
     final class FinalC{}
     static assert(!__traits(compiles, { class D : FinalC{} }));
 
-    scope class ScopeC{}
-    static assert(!__traits(compiles, { auto  sc = new ScopeC(); }));
-    static assert( __traits(compiles, { scope sc = new ScopeC(); }));
```

Comment:
Hum sorry, this deprecation has actually been reverted (in stable), so it's probably better to wait for the merge and revert this.

Reply:
Reverted the changes to this file to unblock this PR.  This test will be fine once `TEST_OUTPUT` implies no ouput because the deprecation warnings will disappear with the stable merge.